### DISCUSSION TITLE
Ocultar historico do emblema

### DISCUSSION THREAD

**Discussion by IvanilsoDaSilva** [community/community#103303] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

<img width="333" alt="Capturar" src="https://github.com/community/community/assets/66145206/059f791c-163e-44cd-9995-8d1810b55d0f">

Como oculto esse historico em baixo do emblema?

**Comment by queenofcorgis** [comment]
Hi @IvanilsoDaSilva , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
